docs/content/en/docs/fleet-manager/rollout/canary.md

```diff
@@ -0,0 +1,410 @@
+    - The `webhooks` provide an extensibility mechanism for the analysis procedures. In this configuration, webhooks communicate with the testloader to generate test traffic for the healthchecks.
+- The `trafficRouting` configuration specifies how traffic will be shifted to the canary deployment during the rollout process. 
+    - The `maxWeight` parameter defines the maximum percentage of traffic that can be routed to the canary before promotion. 
+    - `stepWeight` determines the incremental amount by which traffic will be increased after each successful analysis iteration, allowing the canary to be validated under a gradually growing proportion of real-world load. 
+    - The `gateways` and `host` represent the ingress points for external and internal service traffic, respectively.
+- The `rolloutStatus` section displays the actual processing status of rollout within the fleet.
+
+About a minute after submitting this configuration, you can check the rollout status by running the following command:
+
+```conole
+k get canary -n webapp --kubeconfig=/root/.kube/kurator-member1.config
+
+NAME      STATUS        WEIGHT   LASTTRANSITIONTIME
+backend   Initialized   0        2024-01-11T02:40:40Z
+```
+
+If the status shows as `Initialized`, it means the initialization of rollout process has completed successfully.
+
+
+**Notes**: In the above configuration, we set the `kustomization.interval` to 0s. This disables Fluxcd's periodic synchronization of configurations between the local mirror and cluster. The reason is that Flagger needs to modify the replica counts in Deployments to complete its initialization process. If you are uncertain whether the replicas for all applications in your deployments are set to zero, it is recommended to also set `kustomization.interval` to 0s.
+
+### Trigger Rollout
+
+A Canary Deployment can be triggered by either updating the container image referenced in the git repository configuration, or directly updating the image of the deployment resource locally in the Kubernetes cluster.
+
+Review the results:
+
+```console
+k get canary -n webapp -w --kubeconfig=/root/.kube/kurator-member1.config
+
+NAME      STATUS        WEIGHT   LASTTRANSITIONTIME
+backend   Initialized   0        2024-01-11T02:40:40Z
+backend   Progressing   0        2024-01-11T09:01:40Z
+backend   Progressing   10       2024-01-11T09:03:10Z
+backend   Progressing   10       2024-01-11T09:04:40Z
+backend   Progressing   20       2024-01-11T09:06:10Z
+backend   Progressing   30       2024-01-11T09:07:40Z
+backend   Progressing   40       2024-01-11T09:09:10Z
+backend   Progressing   50       2024-01-11T09:10:40Z
+backend   Promoting     0        2024-01-11T09:12:10Z
+backend   Finalising    0        2024-01-11T09:13:40Z
+backend   Succeeded     0        2024-01-11T09:15:10Z
```

Comment:
If one does not have knowledge of `canary`, he can not understand the meaning of weight here. 

I would suggest you can draw a picture showing weight transition between the two versions